computer-use-demo/computer_use_demo/loop.py

```diff
@@ -74,71 +83,120 @@ async def sampling_loop(
     api_key: str,
     only_n_most_recent_images: int | None = None,
     max_tokens: int = 4096,
+    region: str = "us-central1",  # Region required for Vertex API
```

Comment:
* We want to keep this as agnostic as possible to platform, so having a vertex-only region param isn't desirable
* as documented, the SDK already respects `CLOUD_ML_REGION`